Agents/PubChemAgent/pubchemagent/stack-manager-input-config/pubchem-agent.json

```diff
@@ -0,0 +1,24 @@
+{
+    "ServiceSpec": {
+        "Name": "pubchem-agent",
+        "TaskTemplate": {
+            "ContainerSpec": {
+                "Image": "ghcr.io/cambridge-cares/pubchem_agent:1.0.0",
+                "Env": [
+                    "NAMESPACE=ontospecies"
+                ],
+                "Configs": [
+                    {
+                        "ConfigName": "blazegraph"
+                    }
+                ]
+            }
+        }
+    },
+    "endpoints": {
+        "rest": {
+            "url": "http://localhost:5000",
+            "externalPath": ""
```

Comment:
No external path? That doesn't sound like a good idea to me - that's only asking for clashes... How about the natural choice of `/pubchem-agent/`?

Reply:
I added the external path